src/main/java/com/eliotlash/molang/lexer/Lexer.java

```diff
@@ -79,6 +85,12 @@ private void scanToken() {
 		}
 	}
 
+
+	private void eatString() {
+		while (advance() != '\'') {}
```

Comment:
We should add some error handling for when you're missing a closing quote. Also, should we support double quotes?

Reply:
Added that in latest commit